examples/data_sharing.tex

```diff
@@ -0,0 +1,259 @@
+\documentclass[a4paper]{article}
+\usepackage{../data}
+
+%%%%% Now we input our variable terms as glossary entries %%%%%
+
+\newglossaryentry{type}{
+	name={Company},
+	first={the ``Company''},
+	text={Company},
+	description={is a commercial undertaking}
+}
+
+\newglossaryentry{undertaking}{
+	name={King of the Eagles},
+	first={King of the Eagles, hereafter `\glsentrytext{type}'},
+	text={the \glsentrytext{type}},
+	description={are Big birds. Live on a hill}
+}
+
+\newglossaryentry{processor}{
+	name={Mirkwood Elves},
+	first={Mirkwood Elves, hereafter `the Processor'},
+	text={Data Processor},
+	description={are the Elves of the Noldor Line}
+}
+
+\newcommand{\jurisdiction}{Endor}
+
+\newcommand{\appealBody}{Galadri\"el}
+
+\newcommand{\services}{Orc-slaying}
+
+\newcommand{\companyAddress}{123 Eirie Road}
+
+
+\newcommand{\processorAddress}{Third Tree, Mirkwood}
+
+\newcommand{\processorEmail}{elves@place.mw}
+
+\title{Data Sharing Agreement between \Glsentrytext{undertaking} \& \Glsentrytext{processor}}
+
+\begin{document}
+
+\begin{titlepage}
+
+\centering
+
+\vspace{0.4\textheight}
+
+\vspace{3cm}
+
+\scshape{\Glsfmtname{undertaking}}
```

Comment:
@passamo9 is [small-caps](https://en.wikipedia.org/wiki/Small_caps) ok here (and below), as opposed to all capital cases?

Reply:
That looks great!
I'm sorry for the delay, dealing with some personal issues in the past week.

Reply:
> That looks great!

you mean the small-caps?

did you see my comment above starting with "@passamo9 please check ..."? (just to make sure since the UI doesn't appear to stand out.)